pymongo/topology.py

```diff
@@ -213,7 +213,9 @@ def _select_servers_loop(self, selector, timeout, address):
             # No suitable servers.
             if timeout == 0 or now > end_time:
                 raise ServerSelectionTimeoutError(
-                    self._error_message(selector))
+                    "%s, Timeout: %ss, Topology Description: %s" %
+                    (self._error_message(selector), str(timeout),
+                     repr(self.description)))
```

Comment:
nit: it would be cleaner to remove the str() and repr() calls and rely on the format string (`%s` for str() and `%r` for repr()):
```
                    "%s, Timeout: %ss, Topology Description: %r" %
                    (self._error_message(selector), timeout, self.description))
```